docs/DeveloperGuide.md

```diff
@@ -420,9 +426,14 @@ Use case ends.
 * 2a. The financial consultant enters invalid transaction details or an invalid client index.
     * 2a1. Clientell shows an error message.
     * Use case ends.
+* 2c. The financial consultant inputs an amount that is too large to be handled.
```

Comment:
I think instead of too large to be handled can say not within range? Because there are negative numbers out of range also